### PR TITLE
test to confirm #268

### PR DESCRIPTION
**Pull request by dcrosta**
This confirms the issue reported in #268.

I can work on a patch, but I thought it best to get the test to confirm it in so that we all see it.

### CODE REVIEW

**Comment by dcrosta** [comment]
I should note, credit for the test goes to @gareth-lloyd -- I only modified it minimally to make it reproducible (dropping the collections before doing the test)

File: tests/document.py [hunk 1]
```diff
@@ -2345,6 +2345,31 @@ class DocumentTest(unittest.TestCase):
...
+
+        class User2(Document):
+            name = StringField()
+            email = StringField(primary_key=True)
+        User2.drop_collection()
+
+        User2.objects.create(name='geoff', email='a@b.com')
+
+        u2 = User2(name='frank', email='a@b.com')
+        import pdb
```
Comment (colinhowe) [inline tests/document.py]: I think you may have committed the PDB stuff by accident

Comment (dcrosta) [inline tests/document.py]: I sure did. Thanks for the catch.

**Comment by dcrosta** [comment]
oops. got confused with branches. will reissue pull request

**Pull request merged**